docs/technical-reference/api/manifest-cr.md

```diff
@@ -87,7 +87,16 @@ The resource is the default data that should be initialized for the module and i
 
 ### **.status**
 
-The Manifest CR status is an unmodified version of the [declarative status](../../../internal/declarative/README.md#resource-tracking), so the tracking process of the library applies. There is no custom API for this.
+The status of the Manifest Custom Resource (CR) is a direct, unaltered reflection of the [declarative status](../../../internal/declarative/README.md#resource-tracking), which means it follows the tracking process implemented by the internal library. There isn't a custom API specifically designed for this status tracking. Instead, the existing mechanisms provided by the library are utilized to monitor and update the status of the Manifest CR.  
```

Comment:
thb, the first part is hard to understand for me (generated by GPT?), and we will soon remove the declarative pattern, there is no need to align this declarative status. I suggest removing the first paragraph.